src/Components/NewContact/index.js

```diff
@@ -1,11 +1,111 @@
+        lastName: "",
+        phone: "",
+        email: "",
+        avatar: "",
+    });
+   
+    const handleInputChange = (e) => {
+        setState({
+            ...state,
+            [e.target.name]: e.target.value,
+        });
+    };
+
+    const dontSetNewContact = (e) => {
+        e.preventDefault();
+        openComponent();
+    }
+
+    const sendPicToStorage = (e) => {
+        const picNameId = uuidv4()
+        const imageRef = ref(fireStorage, `contactAvatars/${picNameId}`);
+            uploadBytes( imageRef, e.target[4].files[0] ).then(() => {
+            const imageListRef = ref(fireStorage, `contactAvatars/`);
+            listAll(imageListRef).then((response) => {
+                response.items.forEach((item) => {
+                    if (item.name === picNameId) {
+                        getDownloadURL(item).then((uploadPicUrl) => {
+                        setState({
+                                ...state,
+                                id:uuidv4(),
+                                avatar: uploadPicUrl,
+                            });
+                        });
+                    };
+                });
+            })
+        });
+    };
+
+    const handleSetContact = (e) => {
+        e.preventDefault();
+      if(e.target[4].files[0]){
+          sendPicToStorage(e)
+      }
+        setContacts(state);
+        getContactFromBase();
+        openComponent();
+    };
 
-const NewContact = () => {
     return (
-        <div className="newContact">
-            New contact
-        </div>
+        <form onSubmit={handleSetContact} className={styles.newContact}>
+            <label className={styles.label}>
+                Name: 
+                <input
+                    type="text"
+                    name="firstName"
+                    value={state.firstName}
+                    onChange={handleInputChange}
+                />
+            </label>
+            <label>
+                Surname: 
+                <input 
+                    type="text" 
+                    name="lastName"
+                    value={state.lastName}
+                    onChange={handleInputChange}
+                />
+            </label>
+            <label>
+                Phone:
+                <input 
+                type="text"
+                name="phone"
+                value={state.phone} 
+                onChange={handleInputChange}
+            />
+            </label>
+            <label>
+                E-mail:
+                <input 
+                    type="email" 
+                    name="email" 
+                    value={state.email}
+                    onChange={handleInputChange}
+                />
+            </label>
+            <label>
+                Get photo 
+                <input type="file" name="avatar"/>
+            </label>
+            <button type="submit">add contact</button>
+            <button onClick={(e)=>dontSetNewContact(e)}>go back</button>
```

Comment:
please set spaces before and after "=>" sign, and you could just write `<button onClick={dontSetNewContact}>`